app/mailers/admin_mailer.rb

```diff
@@ -1,6 +1,6 @@
 class AdminMailer < ActionMailer::Base
   add_template_helper(ApplicationHelper)
-  default from: 'no-reply@reservations.app'
+  default from: "no-reply@#{ ActionMailer::Base.default_url_options[:host] }"
```

Comment:
It might not be routable mail, is the thing; but yea that's kinda the point. :stuck_out_tongue: